src/Components/Lists/InventorySummaryList/InventorySummaryList.js

```diff
@@ -0,0 +1,61 @@
+// Import Components
+import React from 'react';
+import PropTypes from 'prop-types';
+import EquipmentTypeSummaryCard from '../../Cards/InventoryTypeSummaryCard/EquipmentTypeSummaryCard';
+import { InventorySummaryResponse } from '../../../ResponseBody';
+import { MESSAGE } from '../../../Constants';
+
+// Import Stylings
+import './InventorySummaryList.css';
+
+// Import Icons
```

Comment:
Addressed! I just put their as to sort out if in the future, we need to put icons in or not. But I think it can be misleading and not look good so i have remove this empty comments in everywhere.